rpc/documentation/api.md

```diff
@@ -169,6 +169,30 @@ synchronizes the wallet to the consensus server if it was previously loaded.
 
 ___
 
+#### `CreateWatchingOnlyWallet`
```

Comment:
Add an anchor link to this under the WalletLoaderService documentation (starts at line 84).